pkg/controller/import-controller.go

```diff
@@ -440,6 +442,11 @@ func (r *ImportReconciler) createImportEnvVar(pvc *corev1.PersistentVolumeClaim)
 		if err != nil {
 			return nil, err
 		}
+		fsOverhead, err := GetFilesystemOverhead(r.uncachedClient, pvc)
```

Comment:
Any particular reason to use the uncached client here? I see the cached client being used in the uploadServer.

Reply:
Nope, missed it. thanks